simpledisplay.d

```diff
@@ -7257,6 +7252,17 @@ version(Windows) {
 			SetWindowTextA(hwnd, toStringz(title));
 		}
 
+		string getTitle() {
+			auto len = GetWindowTextLengthA(hwnd);
+			if (!len)
+				return null;
+			char[] buffer = new char[len];
+			auto len2 = GetWindowTextA(hwnd, buffer.ptr, buffer.length);
```

Comment:
just a note but we should be using the W functions like all the time. i'll fis this myself tho

Reply:
https://github.com/adamdruppe/arsd/commit/ecf04589d0974160e64bd9e01b301be5fca07315